packages/diagrams/lib/src/align.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'diagram_step.dart';
+
+class AlignDiagram extends StatelessWidget implements DiagramMetadata {
+  const AlignDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+//    List<Widget> diagramChildren;
+    const Widget logo = FlutterLogo(size: 60);
+    const Icon origin = Icon(
+      Icons.fiber_manual_record,
```

Comment:
I think I'd use something like "gps_fixed" here instead of just a dot.

Also, for the Fractional Offset Origin diagram, the dot doesn't appear on the upper corner of the container, it is inside of the padding, so it's hard to know that the origin is the top left of the container.